pkgs/servers/nosql/mongodb/6.0.nix

```diff
@@ -9,6 +9,10 @@ buildMongoDB {
   version = "6.0.13";
   sha256 = "sha256-BD3XrTdv4sCa3h37o1A2s3/R0R8zHiR59a4pY0RxLGU=";
   patches = [
+    # Patches a bug that it couldn't build MongoDB 6.0 on gcc 13 because a include in ctype.h was missing
+    # PR at https://github.com/NixOS/nixpkgs/pull/293556
```

Comment:
```suggestion
```
having a link that points to this PR is not that useful

Reply:
Then I don't get the point why @drupol was suggesting that?

Reply:
Having a comment on top of a patch is often required when introducing a patch. I guess a link to the PR means that the patch should be removed at the next bump?

Reply:
When we will package the next version 7.0 it will go into into a different file called 7.0.nix. This means that presumably this patch will be used for the 6.0 for so long as the 6.0 gets deprecated

Reply:
We want to link an *upstream* if possible. Linking this PR where we introduced it, is pointless here.

Reply:
There is no upstream patch. The patchfile is based on this terminal output when building the currently broken software on master.
![grafik](https://github.com/NixOS/nixpkgs/assets/11199213/45589519-5063-4de6-acc7-c6fb12c59d67)